src/render/canvas/canvas-renderer.ts

```diff
@@ -146,12 +146,11 @@ export class CanvasRenderer extends Renderer {
 
     renderTextWithLetterSpacing(text: TextBounds, letterSpacing: number, baseline: number): void {
         if (letterSpacing === 0) {
-            this.ctx.fillText(text.text, text.bounds.left, text.bounds.top + baseline);
+            this.ctx.fillText(text.text, text.bounds.left, text.bounds.top + text.bounds.height - baseline);
         } else {
             const letters = segmentGraphemes(text.text);
             letters.reduce((left, letter) => {
-                this.ctx.fillText(letter, left, text.bounds.top + baseline);
-
+                this.ctx.fillText(text.text, text.bounds.left, text.bounds.top + text.bounds.height - baseline);
```

Comment:
I am just testing this out on my current project. Should the first 2 parameters here not stay as `letter` and `left`?